apps/interactiveevents/templates/a4_candy_interactive_events/module_detail.html

```diff
@@ -1,18 +1,62 @@
 {% extends extends %}
-{% load i18n discovery_tags ckeditor_tags wagtailcore_tags %}
+{% load i18n discovery_tags ckeditor_tags wagtailcore_tags contrib_tags %}
 
 {% block phase_content %}
-    <div class="bg--light">
-        <div class="container">
-            <div class="col-lg-10 offset-lg-1 u-bg-body">
-                <h3>{{ module.name }}</h3>
-                <div class="alert" role="alert">
-                    <i class="fas fa-info-circle" aria-hidden="true"></i>
+    <div class="container">
+        <div class="col-12">
+            <div class="alert" role="alert">
+                <i class="fas fa-info-circle" aria-hidden="true"></i>
                 {% trans 'During this event you can hand in questions with your smartphone' %}
+            </div>
+        </div>
+        <div class="row mx-md-4 my-4 justify-content-center">
```

Comment:
x-axis margins on rows shouldn't be needed as all designs are based on 12 cols so if it lookas too wide, then it probably needs to be less columns